src/assets/scss/custom.scss

```diff
@@ -2,7 +2,7 @@ $primary: #404F65;
 $secondary: #D65F30;
 $success: #219653;
 $info: #3144EC;
-$light: #CFDBE1;
+$light: #EFF0F6;
```

Comment:
If we can apply this just for single datasets view box border could be nice. Because now browse page accordion color also changed

Reply:
We cannot, is the issue.

Reply:
can we introduce a new color for the single datasets view? for example `$light-alternative: #EFF0F6 `and have `$light: #CFDBE1`